python/lsst/ctrl/provenance/dc3.py

```diff
@@ -172,7 +172,7 @@ def initProdRun(self):
         run offset to this run.
         """
         if self._roffset is not None:
-            raise lsst.pex.exceptions.Exception("runId appears to already be registered")
+            raise lsst.pex.exceptions.RuntimeError("runId appears to already be registered")
```

Comment:
I don't think we care but these should all be a normal python RuntimeError and not a pex.exception.